src/passes/Inlining.cpp

```diff
@@ -539,43 +544,28 @@ struct FunctionSplitter {
   // (like limitations on which functions can be inlined into in each iteration,
   // the number of iterations, etc.). Therefore this function will only find out
   // if we *can* split, but not actually do any splitting.
-  bool canSplit(Function* func) {
-    if (!canHandleParams(func)) {
-      return false;
-    }
-
-    // Check if we've processed this input before.
-    auto iter = splits.find(func->name);
-    if (iter != splits.end()) {
-      return iter->second.splitKind != Split::Kind::None;
-    }
-
-    // The default value of split.splittable is false, so if we fail we just
-    // need to return false from this function. If, on the other hand, we can
-    // split, then we will update this split info accordingly.
-    auto& split = splits[func->name];
-
+  InliningMode getSplitDrivenInliningMode(Function* func) {
```

Comment:
The comment above needs updating, in particular "returns true" is no longer accurate as the return type is changed.

Also, how about removing "Driven" from the name? I think it's clearer without it.

Reply:
I put the funky name because getSplitIniningMode may return a non-split inlining mode in the  next PR (i.e. it may return Direct). 
If you don't think that would be confusing, I can do the rename.

Reply:
I'm not sure without seeing the next PR, but perhaps let's rename it here and then rename it again in the next PR? Hopefully we can find a better name at that time - it sounds like the purpose of the function might be changing.

Reply:
Done.